addons/radar/XEH_preInit.sqf

```diff
@@ -197,6 +215,15 @@ private _curCat = localize "STR_dui_cat_compass";
     ,false
 ] call CBA_fnc_addSetting;
 
+[
+    "diwako_dui_compass_icon_scale_crew"
```

Comment:
Please use the QGVAR macro

Reply:
Only these or all option varnames in the file?

Reply:
For the newly added ones. The reason why not all variable names use the GVAR macros are for backwards compatibility before this was a cba template mod.
All new settings should use the macros now.

Reply:
Replaced, now new global vars are using macro